easybuild/easyconfigs/t/TuiView/TuiView-1.1.11-gimkl-2017a-GLib-2.52.0-GEOS-3.5.1-Python-2.7.13.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'TuiView'
+version = '1.1.11'
+
+homepage = "https://tuiview.org/"
+description = """TuiView is a lightweight raster GIS with powerful raster attribute table manipulation
+ abilities."""
+
+toolchain = {'name': 'gimkl', 'version': '2017a'}
+
+source_urls = ['https://bitbucket.org/chchrsc/tuiview/downloads/']
+sources = [SOURCE_TAR_GZ]
+
+glib_version = '2.52.0'
+geos_version = '3.5.1'
+python_version = '2.7.13'
+glibsuffix = '-GLib-%s' % glib_version
+geossuffix = '-GEOS-%s' % geos_version
+pysuffix = '-Python-%s' % python_version
+versionsuffix = '%s%s%s' % (glibsuffix, geossuffix, pysuffix)
```

Comment:
Please use the new `Python` version support and move this section to immediately below `version`.